protocol/0007-POSN-non_core_positions_api.md

```diff
@@ -45,9 +45,19 @@ The API is expected to expose:
 ### Position
 
 - Open volume (this is a core API)
-- Unrealised P&L (method = averaged)
-- Realised P&L (method = averaged)
 - Open volume average entry price (method = averaged)
+- Unrealised P&L (method = averaged)
+- Realised P&L for the market for the key (method = averaged)
```

Comment:
Everything is per market and per key, isn't it? So that should probably go on line `46`. Also if the method is always = averaged then let's say it ones upfront, not repeat.